test/packages/custom_logs/manifest.yml

```diff
@@ -0,0 +1,46 @@
+format_version: 1.0.0
+name: custom_logs
+title: Custom Logs
+description: >-
+  Collect custom logs with Elastic Agent.
+type: input
+version: 1.2.3
+release: ga
+license: basic
+categories:
+  - custom
+policy_templates:
```

Comment:
Do policy templates make sense for input packages? It is really convenient that we can reuse the same logic as we already have so I'm not necessarily asking to change anything here but have a discussion. 

* Can an input package have more then one policy_templates?
* Does an input package need all features of policy_templates?

Reply:
> Does an input package need all features of policy_templates?

It looks like we can improve on this. These are the properties available in integration's `policy_template`:
* name
* title
* categories
* description
* data_streams
* inputs
* multiple (?)
* icons
* screenshot

Clearly, we don't need all of them, so I assumed that this `policy_template` (a different one in terms of spec) will contain a subset (or a different set) of properties. If you consider a different name for `policy_template`, do you have any suggestions? maybe an `input_template`?

> Can an input package have more then one policy_templates?

Good question, but I believe that the answer might be the same as for `policy_templates`. In most cases, we will need one. What should we do in the future if we scope this for one only :)?

Reply:
Does the spec allow to use the same name but for example if it is defined as input package, to not allow certain params?